bot.py

```diff
@@ -3,8 +3,14 @@
 from discord.ext import commands
 from datetime import datetime
 import os
+import configuration
+import ensure_path
+if os.getcwd() != ensure_path.send_path():
+        os.chdir(ensure_path.send_path())
```

Comment:
What does this do?

Reply:
Makes sure the current working dir is correct